aws-redis.yml

```diff
@@ -226,6 +230,12 @@ examples:
   plan_id: c7f64994-a1d9-4e1f-9491-9d8e56bbf146
   provision_params: {"region": "us-east-1"}
   bind_params: {}
+- name: invalid-automatic-failover
+  description: Try to set `automatic_failover_enabled` with `node_count` < 2
+  expected_error: When using automatic failover, there must be at least 2 cache clusters in the replication group
+  plan_id: c7f64994-a1d9-4e1f-9491-9d8e56bbf146
+  provision_params: {"node_count": 1, "automatic_failover_enabled": true}
+  bind_params: {}
```

Comment:
By simply removing these 6 lines, this PR will no longer depend on https://github.com/cloudfoundry/cloud-service-broker/pull/779

Consider doing that if there is an urgency to release this change, and there is reluctancy in merging the feature to support failing examples.

Reply:
```suggestion
```